hosting-a-web-page-on-your-pod.md

```diff
@@ -2,15 +2,20 @@
 
 ### Introduction
 
-This hack enables you to host traditional web pages on your Solid pod.  Solid is 100% backwards compatible with the existing web.  Hosting a webpage on your pod couldn't be easier.  It can either be a normal HTML page.  Or interactive using JavaScript.
+This hack enables you to host traditional web pages on your Solid pod.  Solid is 100% backwards compatible with the existing web.  Hosting a webpage on your pod is free and couldn't be easier.  It can either be a normal HTML page.  Or interactive using JavaScript.
+
+We're going to create public web page hosted on your Solid pod, which can be accessed from:
+
+    https://{yourname.solidpod}/public/mysite/webpage.html
 
```

Comment:
I think I've done this first part!

Reply:
Not sure I understand what you mean with your comment here. Can you please elaborate?

Reply:
Oh sorry, I was adding the text in parts.  I did that first part.  But now I think I've done all.